src/ddmd/expressionsem.d

```diff
@@ -1874,6 +1875,8 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
                 if (!sd.ctor)
                     sd.ctor = sd.searchCtor();
 
+                // If expression is obtained from DCache - value is a literal
+                if (cachedSemantics) goto Lx;
```

Comment:
What's the issue here?

Reply:
I need it to be analyzed as literal even if it looks like a opCall/ctor.